File: platforms/python/examples/scene/camera/bind.py
from fragmentcolor import Camera, Material, Renderer

camera = Camera.perspective(60.0_.to_radians(), 16.0 / 9.0, 0.1, 100.0).look_at([0.0, 1.0, 5.0], [0.0, 0.0, 0.0], [0.0, 1.0, 0.0])

renderer = Renderer()
material = Material.pbr(renderer)
camera.bind(material.shader())

File: platforms/swift/examples/scene/camera/bind.swift
import FragmentColor

let camera = Camera.perspective(60.0.toRadians(), 16.0 / 9.0, 0.1, 100.0).lookAt([0.0, 1.0, 5.0], [0.0, 0.0, 0.0], [0.0, 1.0, 0.0])

let material = Material.pbr()
camera.bind(material.shader())